pytest/tests/mocknet/mirror.py

```diff
@@ -604,13 +611,43 @@ def filter_hosts(args, traffic_generator, nodes):
     # nearcore-release buildkite and urls in the following format without commit
     # but only with the branch name:
     # https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore/Linux/<branch-name>/neard"
-    update_binaries_parser = subparsers.add_parser(
-        'update-binaries',
-        help=
-        'Update the neard binaries by re-downloading them. The same urls are used.'
-    )
+    update_binaries_parser = subparsers.add_parser('update-binaries',
+                                                   help='''
+        Update the neard binaries by re-downloading them. The same urls are used.
+        If you plan to restart the network multiple times, it is recommended to use
+        URLs that only depend on the branch name. This way, every time you build,
+        you will not need to amend the URL but just run update-binaries.''')
     update_binaries_parser.set_defaults(func=update_binaries_cmd)
 
+    amend_binaries_parsers = subparsers.add_parser('amend-binaries',
+                                                   help='''
+        Add or override the neard URLs by specifying the epoch height or index if you have multiple binaries.
+
+        If the network was started with 2 binaries, the epoch height for the second binary can be randomly assigned
+        on each host. Use caution when updating --epoch-height so that it will not add a binary in between the upgrade
+        window for another binary.''')
+
+    amend_binaries_parsers.add_argument('--neard-binary-url',
+                                        type=str,
+                                        required=True,
+                                        help='URL to the neard binary.')
+    group = amend_binaries_parsers.add_mutually_exclusive_group(required=True)
+    group.add_argument('--epoch-height',
+                       type=int,
+                       help='''
+        The epoch height where this binary will begin to run.
+        If a binary already exists on the host for this epoch height, the old one will be replaced.
+        Otherwise a new binary will be added with this epoch height.
+        ''')
+    group.add_argument('--binary_idx',
```

Comment:
could we make this `--binary-idx` instead of the underscore to keep with the style elsewhere?